lib/Myriad/RPC.pm

```diff
@@ -0,0 +1,98 @@
+package Myriad::RPC;
+
+use strict;
+use warnings;
+
+# VERSION
+
+use Sys::Hostname;
+use Future::AsyncAwait;
+use Object::Pad;
+use Log::Any qw($log);
+
+use Myriad::RPC::Message;
+
+class Myriad::RPC extends Myriad::Notifier;
+
+use experimental qw(signatures);
+
+use utf8;
+
+use Syntax::Keyword::Try;
+
+=encoding utf8;
+
+=head1 SYNOPSIS
+
+=head1 DESCRIPTION
+
+Myriad RPC implementation to serve the requests of the service clients.
+
+=cut
+
+has $redis;
+has $service;
+
+has $group_name;
+
+has $ryu;
+has $rpc_map;
+has $whoami;
+
+method ryu {$ryu}
```

Comment:
Style-wise don't forget the contents of the `{...}` is an entire block of method code. Even for a short example like this it might be nice to put a little bit of space and a terminating semicolon in there:

```perl
method ryy { $ryu; }
```

Similar for the others